src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -153,6 +159,19 @@ public HostOutputMessage HostOutputMessage
             return exceptionInfo != null ? new ExceptionDispatchInfoDelayedException(exceptionInfo) : null;
         }
 
+        private async Task InvokeExceptionFiltersAsync(Exception exception, IFunctionInstance functionInstance, CancellationToken cancellationToken)
+        {
+            var exceptionFilters = GetFilters<IFunctionExceptionFilter>(_extensionRegistry, functionInstance.FunctionDescriptor, functionInstance.JobInstance);
```

Comment:
>functionInstance.JobInstance [](start = 129, length = 28)

currently, JobInstance is lazy; so this could call IJobActivator which could throw; and then we'd miss the exception.